ui/src/app/components/app-router.tsx

```diff
@@ -145,17 +99,12 @@ export const AppRoute = ({
     fp.find(({ allowed }) => !allowed(), guards) || {};
 
   return (
-    <Route exact={exact} path={path}>
+    <CompatRoute {...{ path, exact }}>
       {cond<React.ReactNode>(
-        [redirectPath, () => <Redirect to={redirectPath} />],
+        [redirectPath, () => <Navigate to={redirectPath} />],
         [renderBlocked, () => renderBlocked()],
         () => children
       )}
-    </Route>
+    </CompatRoute>
   );
 };
-
-export const Navigate = ({ to }): React.ReactElement => {
```

Comment:
only used by AuditPageComponent, and we were not using the wrapped functionality (location) so we can use the native version instead